### PR TITLE
ch2 LOs

### PR DESCRIPTION
**Pull request by jonathanbratt**
I thought about including an LO for the R lab part, but that felt... useless. I think if we write lab LOs we should treat them kinda separately from the rest of the chapter.

### CODE REVIEW

File: 02.Rmd [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Statistical Learning
+
+**Learning objectives:**
+
+- Define Statistical Learning.
```
Comment (jonthegeek) [inline 02.Rmd:5]: As we talked about on Slack, this should be a little broader. "Describe how statistical learning can be used to understand data" or something, maybe?

Comment (jonthegeek) [reply 02.Rmd:5]: Oh, it's also possible this LO just goes away, if the other ones cover all the expansion.

File: 02.Rmd [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- Understand the limits on the accuracy of statistical models.
+- Describe the trade-off between more accurate models and more interpretable models.
+- Compare and contrast supervised and unsupervised learning.
+- Compare and contrast regression and classification problems.
+- Measure the goodness of regression model fits.
```
Comment (jonthegeek) [inline 02.Rmd:10]: I haven't read the chapter yet, but are we learning to measure these things, or learning to recognize the importance of measuring these things (or something along those lines)?

**Review by jonthegeek — CHANGES_REQUESTED** [review]
This might not actually require any changes, but let's talk about it.

File: 02.Rmd (hunk 2 above)
Comment (jonathanbratt) [reply 02.Rmd:10]: It's defining actual measures.

File: 02.Rmd (hunk 1 above)
Comment (jonathanbratt) [reply 02.Rmd:5]: yeah, the expansion feels a little gratuitous, so I'll just remove it.

**Review by jonthegeek — APPROVED** [review]
Looks good! I might have changes after I actually read the chapter, but I actually wanted to read these before reading 😁

**Pull request merged**